wear/src/main/AndroidManifest.xml

```diff
@@ -122,6 +134,16 @@
 
             <meta-data android:name="androidx.wear.tiles.PREVIEW"
                 android:resource="@drawable/favorite_entities_tile_example" />
+
+            <meta-data
+                android:name="com.google.android.clockwork.tiles.MULTI_INSTANCES_SUPPORTED"
+                android:value="true" /> <!-- This is supported starting from Wear OS 3 -->
+
+            <meta-data
+                android:name="com.google.android.clockwork.tiles.PROVIDER_CONFIG_ACTION"
+                android:value="ConfigShortcutsTile" />
+            <!-- Having PROVIDER_CONFIG_ACTION here is required for the Shortcut Tile to show
+                 up in the Tile selector when there's already a Shortcut Tile added. -->
```

Comment:
This comment feels unnecessary; it now actually serves as a config action and the previous metadata above makes it clear multiple tiles are supported.

Reply:
Removed in c3d81bf4